### DISCUSSION TITLE
Copy current file path with wl-copy

### DISCUSSION THREAD

**Discussion by omnigenous** [gokcehan/lf#1733] [Q&A] [answered]
Can't get [this](https://github.com/gokcehan/lf/wiki/Tips#ctrlc-to-copy-one-file-path) to work with wl-copy properly

```
cmd copy_to_clipboard %{{
    printf '%s' "$f" | wl-copy
}}
map <c-c> :copy_to_clipboard
```

It does copy file path  to clipboard but it also open prompt(?) at the bottom of lf (see screenshot) that can't be closed with another press of `control+c` but I rather not do that if possible.

![2024-05-30_12-26-01](https://github.com/gokcehan/lf/assets/154441415/51fbc6a6-3911-422d-b127-6bfee41f8286)

**Comment by DusanLesan** [comment, marked as answer]
That does not look right to me. Doc says: `%  shell-pipe      shell command running with the UI`
Please try:
```
cmd copy_to_clipboard ${{
    printf '%s' "$f" | wl-copy
}}
map <c-c> copy_to_clipboard
```

**Reply by omnigenous** [reply]
Yep, replacing `%` with `$` did the trick. Need to read docs and see what it does :D
